code/game/objects/items/weapons/cigs.dm

```diff
@@ -221,26 +221,96 @@ LIGHTERS ARE IN LIGHTERS.DM
 	..()
 
 /obj/item/clothing/mask/cigarette/syndicate
+	desc = "A roll of tobacco and nicotine in a cool but suspicious black and red paper. Smells... weird, but it makes your nose feel better."
+	item_state = "cigoff_syndicate"
+	icon_state = "cigoff_syndicate"
+	icon_on = "cigon_syndicate"
+	icon_off = "cigoff_syndicate"
+	type_butt = /obj/item/cigbutt/syndicate
 	list_reagents = list("nicotine" = 40, "omnizine" = 20)
 
 /obj/item/clothing/mask/cigarette/medical_marijuana
+	desc = "A roll of marijuana in a medical white paper with a blue band. Smells... calming."
+	item_state = "cigoff_med"
+	icon_state = "cigoff_med"
+	icon_on = "cigon_med"
+	icon_off = "cigoff_med"
+	type_butt = /obj/item/cigbutt/med
 	list_reagents = list("thc" = 40, "cbd" = 20)
 
 /obj/item/clothing/mask/cigarette/robustgold
+	desc = "A premium roll of of tobacco and nicotine in a fancy gold paper. Smells a bit like wealth... and cancer."
+	item_state = "cigoff_gold"
+	icon_state = "cigoff_gold"
+	icon_on = "cigon_gold"
+	icon_off = "cigoff_gold"
+	type_butt = /obj/item/cigbutt/gold
 	list_reagents = list("nicotine" = 40, "gold" = 1)
+	
+/obj/item/clothing/mask/cigarette/robust
+	desc = "A roll of of tobacco and nicotine in a white paper with a red filter. Smells... robust. Nothing special about this one otherwise."
+	item_state = "cigoff_robust"
+	icon_state = "cigoff_robust"
+	icon_on = "cigon_robust"
+	icon_off = "cigoff_robust"
+	type_butt = /obj/item/cigbutt/robust
+	list_reagents = list("nicotine" = 40)
 
 /obj/item/clothing/mask/cigarette/shadyjims
+	desc = "A surprisingly thin roll of of tobacco and nicotine in a rather dull gray paper. Smells... like being fit feels...and a bit like battery acid."
+	item_state = "cigoff_shadyjims"
+	icon_state = "cigoff_shadyjims"	
+	icon_on = "cigon_shadyjims"
+	icon_off = "cigoff_shadyjims"
+	type_butt = /obj/item/cigbutt/shadyjims
 	list_reagents = list("nicotine" = 40, "lipolicide" = 7.5, "ammonia" = 2, "atrazine" = 1, "toxin" = 1.5)
 
+/obj/item/clothing/mask/cigarette/dromedaryco
+	desc = "An unfiltered roll of of tobacco and nicotine in a normal white paper. Smells rather strong."
+	item_state = "cigoff_dromedaryco"
+	icon_state = "cigoff_dromedaryco"
+	icon_on = "cigon_dromedaryco"
+	icon_off = "cigoff_dromedaryco"
+	type_butt = /obj/item/cigbutt/dromedaryco
+	list_reagents = list("nicotine" = 50)
+
+/obj/item/clothing/mask/cigarette/carp
+	desc = "A roll of of tobacco and nicotine in a normal white paper with a purple filter. Smells like a space carp."
+	item_state = "cigoff_carp"
+	icon_state = "cigoff_carp"
+	icon_on = "cigon_carp"
+	icon_off = "cigoff_carp"
+	type_butt = /obj/item/cigbutt/carp
+	list_reagents = list("nicotine" = 40, "carpotoxin" = 1)
+
+/obj/item/clothing/mask/cigarette/midori
+	desc = "A roll of of tobacco and nicotine in a normal white paper with a yellow filter. Smells rather funny but otherwise nothing special."
```

Comment:
```suggestion
	desc = "A roll of tobacco and nicotine in a normal white paper with a yellow filter. Smells rather funny but otherwise nothing special."
```